types/i18next-express-middleware/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: http://i18next.com/
 // Definitions by: Cyril Schumacher <https://github.com/cyrilschumacher>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
```

Comment:
@GiedriusGrabauskas Though current build succeeds, definitions are still incorrect because i18next-express-middleware and i18next-sprintf-postprocessor modules have `"i18next": ["i18next/v2"]` path mapping so they have to use old i18next/v2 definitions which is compatible with TypeScript pre-2.1 and thus specifying `// TypeScript Version: 2.3` header is not required. But because i18next-* modules use `/// <reference types=".." />` syntax to reference i18next definitions the above path mapping is ignored (see last sentence in [I want to update a package to a new major version](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version)). So here we have two options:
1. Remove `"i18next": ["i18next/v2"]` path mapping from i18next-* modules tsconfig.json and add `// TypeScript Version: 2.3` header to them. You may need to make changes to all these modules definitions to make them compatible with i18next 8.4 definitions.
2. Add `"i18next": ["i18next/v2"]` path mapping to all these modules. Remove added `// TypeScript Version: 2.3` header.
Also in both cases you need to  remove all `/// <reference types=".." />` or in some cases replace them with `import`.